src/families/bitcoin/speculos-deviceActions.js

```diff
@@ -42,10 +42,17 @@ const acceptTransaction: DeviceAction<Transaction, {}> = ({
   }
 
   if (s.screen === "fees") {
-    let expectedText = formatCurrencyUnit(account.unit, status.estimatedFees, {
-      showCode: true,
-      disableRounding: true,
-    }).replace(/\s/g, " ");
+    let expectedText = formatCurrencyUnit(
+      {
+        ...account.unit,
+        code: account.currency.deviceTicker || account.unit.code,
```

Comment:
nice catch. i wonder if the formatter hsouldn't have a flag to say you want the deviceTicker, but this will definitely do it for now! in future bunch of this code will move somewhere else because we're not yet done on this "deviceActions" here, as you may see the code of this file is pretty spaghetti but we'll figure out a higher level utility for this & more reusable for other apps ;)